rplugin/python3/nvim_ipy/__init__.py

```diff
@@ -273,6 +273,78 @@ def ipy_run(self, args):
                 # TODO: if this is long, open separate window
                 self.append_outbuf(p['text'])
 
+    @neovim.function("IPyRunCell")
+    def ipy_run_cell(self, silent):
+        '''Runs all the code in between two cell separators
+        
+        taken from https://github.com/wmvanvliet/vim-ipython'''
+        cur_buf = self.vim.current.buffer
+        (cur_line, cur_col) = self.vim.current.window.cursor
+        cur_line -= 1
+        def is_cell_separator(line):
```

Comment:
This can probably be implemented much simpler vimL-side, where we can use ordinary regex `search()`. Instead we should probably have a Python function IPyRunLines which takes a line range.